### PR TITLE
fix(curriculum): change the word "property" to "attribute"

### PR DESCRIPTION
**Pull request by sidemt**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

Affected page:
(New) Responsive Web Design > Learn Basic CSS by Building a Cafe Menu > Step 77
https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-77

I think we should call `class="bottom-line"` an attribute.

### CODE REVIEW

**Comment by RandellDawson** [comment]
I am thinking maybe it should be reworded to something like the following since it is an attribute/value reference.
> You should add the `class` attribute with the value `bottom-line`.

**Review by Sboonny — DISMISSED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

Edit: whoops, didn't notice RendellDawson comment

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f4701b942c824109626c3d8.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Now add the `bottom-line` class to the second `hr` element so the styling is app
 
 # --hints--
 
-You should apply the `class="bottom-line"` property.
+You should apply the `class="bottom-line"` attribute.
```
Comment (nhcarrigan) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f4701b942c824109626c3d8.md:14]:
```suggestion
You should add the `class` attribute with the value `bottom-line`.
```

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Comment by sidemt** [comment]
Thanks @RandellDawson @Sembauke @nhcarrigan ! Updated.

**Review by naomi-lgbt — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**